cmd/check.go

```diff
@@ -88,7 +90,7 @@ func init() {
 	flags.StringVar(&checkflags.CatalogSourceNamespace, "catalogsourcenamespace", "openshift-marketplace",
 		"specifies the namespace where the catalogsource exists")
 	flags.StringSliceVar(&checkflags.AuditPlan, "auditplan", defaultAuditPlan, "audit plan is the ordered list of operator test functions to be called during a capability audit.")
-	flags.BoolVar(&checkflags.ListPackages, "list-packages", false, "list packages in the catalog")
-	flags.StringSliceVar(&checkflags.FilterPackages, "filter-packages", []string{}, "a list of package(s) which limits audits and/or other flag(s) output")
-	flags.BoolVar(&checkflags.AllInstallModes, "all-installmodes", false, "when set, all install modes supported by an operator will be tested")
+	flags.BoolVar(&checkflags.ListPackages, "list-packages", false, "list packages in the catalog.")
```

Comment:
Should these have periods at the end? I wouldn't think so...

Reply:
I've removed them.